pkg/pb/gc.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package gc;
+
+import "google/protobuf/timestamp.proto";
+
+service GarbageCollection {
+    rpc Delete(DeleteRequest) returns (DeleteResponse);
+}
+
+message BloomFilter {
```

Comment:
Why not just `Filter` as we defined it in the design doc?

Reply:
I don't know :) Changed.